aten/src/ATen/native/cuda/UpSampleBicubic2d.cu

```diff
@@ -28,18 +338,21 @@ Tensor& upsample_bicubic2d_backward_out_cuda(
     IntArrayRef output_size,
     IntArrayRef input_size,
     bool align_corners) {
-    return at::legacy::th::_thnn_upsample_bicubic2d_backward_out(
-        grad_input, grad_output, output_size, input_size, align_corners);
+  upsample_bicubic2d_backward_out_cuda_template(
+      grad_input, grad_output, output_size, input_size, align_corners);
+  return grad_input;
 }
 
 Tensor upsample_bicubic2d_backward_cuda(
     const Tensor& grad_output,
     IntArrayRef output_size,
     IntArrayRef input_size,
     bool align_corners) {
-    return at::legacy::th::_thnn_upsample_bicubic2d_backward(
-        grad_output, output_size, input_size, align_corners);
+  Tensor grad_input = at::zeros(input_size, grad_output.options());
```

Comment:
grad_input is set to zero in the template, no need to do it here.

Reply:
ok thanks @ngimel ! I changed `at::zeros` to `empty_like`. let me know if you have another suggestion for it.